tests/api/collections.ts

```diff
@@ -0,0 +1,25 @@
+import { collectionsQuery } from '../queries';
+import axios from 'axios';
+import 'dotenv/config';
+import { handleResponse } from './responseHandlers';
+
+const scanApi = axios.create({
+  baseURL: process.env.GRAPHQL_URL,
```

Comment:
As I can see you don't have this variable in your configuration.

Reply:
done